application-templates/starter-typescript/src/components/channel-details/channel-details.spec.tsx

```diff
@@ -0,0 +1,313 @@
+    });
+  });
+  it('should display a key field validation message if the submitted key value is duplicated', async () => {
+    useMockServerHandlers([
+      fetchChannelDetailsQueryHandler,
+      updateChannelDetailsHandlerWithDuplicateFieldError,
+    ]);
+    renderApp();
+
+    const keyInput: HTMLInputElement = await screen.findByLabelText(
+      /channel key/i
+    );
+
+    fireEvent.change(keyInput, {
+      target: { value: newKey },
+    });
+    expect(keyInput.value).toBe(newKey);
+
+    // updating channel details
+    const saveButton = screen.getByRole('button', { name: /save/i });
+    fireEvent.click(saveButton);
+
+    await screen.findByText(/a channel with this key already exists/i);
+  });
+});
+describe('notifications', () => {
+  it('should render a success notification after an update', async () => {
+    useMockServerHandlers([
+      fetchChannelDetailsQueryHandler,
+      updateChannelDetailsHandler,
+    ]);
+    renderApp();
+
+    const keyInput: HTMLInputElement = await screen.findByLabelText(
+      /channel key/i
+    );
+    expect(keyInput.value).toBe(key);
+
+    fireEvent.change(keyInput, {
+      target: { value: newKey },
+    });
+    expect(keyInput.value).toBe(newKey);
+
+    const rolesSelect = screen.getByRole('combobox', {
+      name: /channel roles/i,
+    });
+    fireEvent.focus(rolesSelect);
+    fireEvent.keyDown(rolesSelect, { key: 'ArrowDown' });
+    const inventorySupplyOption = await screen.findByText('InventorySupply');
+
+    inventorySupplyOption.click();
+    expect(screen.getByDisplayValue(/InventorySupply/i)).toBeInTheDocument();
+
+    // updating channel details
+    const saveButton = screen.getByRole('button', { name: /save/i });
+    fireEvent.click(saveButton);
+    const notification = await screen.findByRole('alertdialog');
+    within(notification).getByText(/channel .+ updated/i);
+  });
+  it('should render an error notification if fetching channel details resulted in an error', async () => {
+    useMockServerHandlers([fetchChannelDetailsQueryHandlerWithError]);
+    renderApp();
+    await screen.findByText(
+      /please check your connection, the provided channel ID and try again/i
+    );
+  });
+  it('should display an error notification if an update resulted in an unmapped error', async () => {
+    useMockServerHandlers([
+      fetchChannelDetailsQueryHandler,
+      updateChannelDetailsHandlerWithARandomError,
+    ]);
+    renderApp();
+
+    const keyInput = await screen.findByLabelText(/channel key/i);
+
+    // we're firing the input change to enable the save button, the value itself is not relevant
+    fireEvent.change(keyInput, {
+      target: { value: 'not relevant' },
+    });
+
+    // updating channel details
+    const saveButton = screen.getByRole('button', { name: /save/i });
+    fireEvent.click(saveButton);
+
+    const notification = await screen.findByRole('alertdialog');
+    within(notification).getByText(/some fake error message/i);
```

Comment:
It seems your PR was released in [v8.13.0](https://github.com/testing-library/dom-testing-library/pull/1120#issuecomment-1088659836) 🙌 However, the version in our repo resolves to v8.11.2

Reply:
Well, that's actually because we didn't update the lock file.
We don't have `@testing-library/dom` as a direct dependency but as a transitive dependency through `@testing-library/react`.
We have `v12.1.5` of the latter which has a range dependency which would allow for `v8.13.0` of the former to be downloaded when running `yarn up -R @testing-library/dom` (which I guess it's what we did in MC frontend repository).

Anyway, I don't want to make more noise in this regard, so forget about it.